src/main/java/com/github/games647/flexiblelogin/FlexibleLogin.java

```diff
@@ -52,12 +59,7 @@
 
     private Hasher hasher;
 
-    @Inject
-    public FlexibleLogin(Logger logger, PluginContainer pluginContainer, Game game) {
```

Comment:
Passing an instance of these class is useless because it can be created in the main plugin class.